web/ui/react-app/src/pages/targets/TargetLabels.tsx

```diff
@@ -16,7 +15,7 @@ export interface SeriesLabelsProps {
 
 const formatLabels = (labels: Labels): string[] => Object.keys(labels).map(key => `${key}="${labels[key]}"`);
 
-const SeriesLabels: FC<SeriesLabelsProps> = ({ discoveredLabels, labels, idx, scrapePool }: SeriesLabelsProps) => {
+const TargetLabels: FC<TargetLabelsProps> = ({ discoveredLabels, labels, idx, scrapePool }: TargetLabelsProps) => {
```

Comment:
I guess Typescript already know about props type since the component is declared as ```FC<TargetLabelProps>```

Reply:
done, thanks for the catch!